charts/astronomer/templates/_helpers.yaml

```diff
@@ -394,3 +407,26 @@ true
 false
 {{- end }}
 {{- end }}
+
+
+{{ define "houston.jetStreamSSLCertificateDir" -}}
+/etc/houston/jetstream/tls
+{{- end }}
+
+
```

Comment:
```suggestion

{{ define "houston.jetStreamSSLCertificateDir" -}}
/etc/houston/jetstream/tls
{{- end }}

```

Reply:
whats the diff between previous and modified wasn't able to see any diff

Reply:
The diff is whitespace.